app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,126 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light-gray: #777;
+$orange: #F25204;
+$dark-blue: #093D5E;
+$brown: #5C2D22;
+$light-brown: #9E4D0B;
+$tan: #E7CE9F;
+$purple: #800080;
+$white: #fff;
+
+// From adobe https://color.adobe.com/Theme-20-color-theme-5544017/
+$grey: #D9C7C1;
+$yellow: #F2CA50;
+$orange_yellow: #F2994B;
```

Comment:
Color `#F2994B` should be written as `#f2994b`<br>Name of variable `orange_yellow` should be written in all lowercase letters with hyphens instead of underscores